hazelcast/src/main/java/com/hazelcast/map/impl/proxy/MapProxyImpl.java

```diff
@@ -333,6 +334,7 @@ public boolean isLocked(K key) {
 
     @Override
     public Map<K, V> getAll(Set<K> keys) {
+        checkNotNull(keys, NULL_KEYS_ARE_NOT_ALLOWED);
```

Comment:
It is a behavioral chance, but passing in a null 'keys' is an error which should be made visible as soon as possible with a NPE (fail fast).